ci/deb-ppa-upload/docker-compose.yml

```
@@ -5,12 +5,13 @@ services:
     build:
       context: build-context
       dockerfile: ../Dockerfile # relative to build-context
+      target: ${TARGET:-CIRCLE_CI}
       args:
         - user_id=${USER_ID:-0}
         - group_id=${GROUP_ID:-0}
     volumes:
       # Host path is relative to current directory, not build-context
-      - ./../..:/home/ci-user/git-machete
+      - ./../..:/${MOUNT_POINT:-root}/git-machete
```

Uber-nitpick: I'd make the leading `/` a part of the mount point path as well:
```suggestion
      - ./../..:${MOUNT_POINT:-/root}/git-machete
```

---

To be reflected in other files on this PR as well as in #425